### PR TITLE
add softshrink kernel

### PR DESCRIPTION
**Pull request by WindQAQ**
- Address #437 - softshrink kernel https://www.gabormelli.com/RKB/Softshrink_Activation_Function

### CODE REVIEW

File: tensorflow_addons/activations/softshrink.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from tensorflow_addons.utils import keras_utils
+from tensorflow_addons.utils.resource_loader import get_path_to_datafile
+
+_activation_ops_so = tf.load_op_library(
+    get_path_to_datafile("custom_ops/activations/_activation_ops.so"))
+
+
+@keras_utils.register_keras_custom_object
+@tf.function
+def softshrink(x, lower=-1.0, upper=1.0):
```
[the remark this answers is not in the document]
Comment (seanpmorgan) [reply tensorflow_addons/activations/softshrink.py]: Also 0.5 default in PaddlePaddle:
https://github.com/PaddlePaddle/Paddle/blob/master/paddle/fluid/operators/activation_op.cc#L251

[the remark this answers is not in the document]
Comment (WindQAQ) [reply tensorflow_addons/activations/softshrink.py]: Probably no... It's my mistake on both `hardshrink` and `softshrink` though I could not find any research paper about the value `0.5`. Will change them to `0.5` later.

[the remark this answers is not in the document]
Comment (seanpmorgan) [reply tensorflow_addons/activations/softshrink.py]: Sounds good. Also couldn't find any rationale for `0.5` other than framework defaults.

**Comment by wangmorgan** [comment]
can not install tensorflow-addons
pls help

**Comment by wangmorgan** [comment]
ERROR: Could not find a version that satisfies the requirement tensorflow-addons (from versions: none)
ERROR: No matching distribution found for tensorflow-addons

**Comment by seanpmorgan** [comment]
> ERROR: Could not find a version that satisfies the requirement tensorflow-addons (from versions: none)
> ERROR: No matching distribution found for tensorflow-addons

Hi @wangmorgan it's likely that your pip version is too old to find the manylinux2010 pkg. Try running `pip install --upgrade pip` prior to installing

**Pull request merged**